monest/views.py

```diff
@@ -186,6 +186,12 @@ def company_scores(request):
     return JsonResponse([res], safe=False)
 
 
+@api_view(['GET', 'POST'])
+def company_total_scores(request):
+    name = request.query_params['0']
```

Comment:
'0' as parameter name is a bad implementation of existing cold. That's part of the reason we're doing optimization. 
So change it to some thing meaningful, say 'company'